harness/determined/estimator/_estimator_context.py

```diff
@@ -133,3 +133,12 @@ class EstimatorExperimentalContext(_data_layer.DataLayerContext):
 
     def __init__(self, env: det.EnvContext, hvd_config: horovod.HorovodContext) -> None:
         super().__init__(env=env, hvd_config=hvd_config)
+        self._allgather_fn = None  # type: Optional[Callable[[Any], List]]
+
+    def _set_allgather_fn(self, fn: Callable[[Any], List]) -> None:
+        self._allgather_fn = fn
+
+    def allgather_metrics(self, stuff: Any) -> List:
```

Comment:
can we rename the arg from stuff to metrics

Reply:
fixed